PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/hdinsight/spark/ui/SparkSubmissionAdvancedConfigDialog.java

```diff
@@ -0,0 +1,279 @@
+
+        add(sshPasswordUseArtifactRadioButtion,
+                new GridBagConstraints(0, ++displayLayoutCurrentRow,
+                        0, 1,
+                        0, 0,
+                        GridBagConstraints.WEST, GridBagConstraints.NONE,
+                        new Insets(margin / 3, margin * 3, 0, margin), 0, 0));
+
+        add(sshPasswordUsePasswordRadioButton,
+                new GridBagConstraints(0, ++displayLayoutCurrentRow,
+                        1, 1,
+                        0, 0,
+                        GridBagConstraints.WEST, GridBagConstraints.NONE,
+                        new Insets(margin / 3, margin * 3, 0, margin), 0, 0));
+
+        add(sshPasswordUsePasswordField,
+                new GridBagConstraints(1, displayLayoutCurrentRow,
+                        0, 1,
+                        1, 0,
+                        GridBagConstraints.WEST, GridBagConstraints.HORIZONTAL,
+                        new Insets(margin / 3, margin, 0, margin), 0, 0));
+
+        add(sshPasswordUseKeyFileRadioButton,
+                new GridBagConstraints(0, ++displayLayoutCurrentRow,
+                        1, 1,
+                        0, 0,
+                        GridBagConstraints.WEST, GridBagConstraints.NONE,
+                        new Insets(margin / 3, margin * 3, 0, margin), 0, 0));
+
+        add(sshPasswordUseKeyFileTextField,
+                new GridBagConstraints(1, displayLayoutCurrentRow,
+                        0, 1,
+                        1, 0,
+                        GridBagConstraints.WEST, GridBagConstraints.HORIZONTAL,
+                        new Insets(margin / 3, margin, 0, margin), 0, 0));
+
+        sshPasswordUseArtifactRadioButtion.setSelected(true);
+        enableRemoteDebugCheckBox.setSelected(false);
+    }
+
+    private void addOperationPanel() {
+        JPanel operationPanel = new JPanel();
+        operationPanel.setLayout(new FlowLayout());
+
+        JButton okButton = new JButton("Ok");
+        JButton cancelButton = new JButton("Cancel");
+        operationPanel.add(okButton);
+        operationPanel.add(cancelButton);
+
+        okButton.addActionListener(new ActionListener() {
+            @Override
+            public void actionPerformed(ActionEvent e) {
+                onOk();
+            }
+        });
+
+        cancelButton.addActionListener(new ActionListener() {
+            @Override
+            public void actionPerformed(ActionEvent e) {
+                onCancel();
+            }
+        });
+
+        getRootPane().setDefaultButton(cancelButton);
+
+        add(operationPanel,
+                new GridBagConstraints(1, ++displayLayoutCurrentRow,
+                        0, 1,
+                        1, 0,
+                        GridBagConstraints.SOUTHEAST, GridBagConstraints.NONE,
+                        new Insets(0, margin, 0, 0), 0, 0));
+    }
+
+    private void onOk() {
+        dispose();
```

Comment:
I need to discuss with you about what you need to pass. For this checkin, let's get the UI in first.

---
In reply to: [117139713](https://github.com/Microsoft/azure-tools-for-java/pull/455#discussion_r117139713) [](ancestors = 117139713)